api/config/lighthouse.php

```diff
@@ -29,84 +30,113 @@
     |
-    | The guard to use for authenticating GraphQL requests, if needed.
-    | This setting is used whenever Lighthouse looks for an authenticated user, for example in directives
-    | such as `@guard` and when applying the `AttemptAuthentication` middleware.
+    | The guards to use for authenticating GraphQL requests, if needed.
+    | Used in directives such as `@guard` or the `AttemptAuthentication` middleware.
+    | Falls back to the Laravel default if `null`.
     |
     */
 
-    'guard' => 'api',
+    'guards' => ['api'],
 
     /*
     |--------------------------------------------------------------------------
-    | Schema Location
+    | Schema Path
     |--------------------------------------------------------------------------
     |
     | Path to your .graphql schema file.
     | Additional schema files may be imported from within that file.
     |
     */
 
-    'schema' => [
-        'register' => base_path('graphql/schema.graphql'),
-    ],
+    'schema_path' => base_path('graphql/schema.graphql'),
 
     /*
     |--------------------------------------------------------------------------
     | Schema Cache
     |--------------------------------------------------------------------------
     |
     | A large part of schema generation consists of parsing and AST manipulation.
-    | This operation is very expensive, so it is highly recommended to enable
+    | This operation is very expensive, so it is highly recommended enabling
     | caching of the final schema to optimize performance of large schemas.
     |
     */
 
-    'cache' => [
+    'schema_cache' => [
         /*
          * Setting to true enables schema caching.
          */
-        'enable' => env('LIGHTHOUSE_CACHE_ENABLE', env('APP_ENV') !== 'local'),
+        'enable' => env('LIGHTHOUSE_SCHEMA_CACHE_ENABLE', env('APP_ENV') !== 'local'),
+
+        /*
+         * File path to store the lighthouse schema.
+         */
+        'path' => env('LIGHTHOUSE_SCHEMA_CACHE_PATH', base_path('bootstrap/cache/lighthouse-schema.php')),
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Cache Directive Tags
+    |--------------------------------------------------------------------------
+    |
+    | Should the `@cache` directive use a tagged cache?
+    |
+    */
+    'cache_directive_tags' => false,
+
+    /*
+    |--------------------------------------------------------------------------
+    | Query Cache
+    |--------------------------------------------------------------------------
+    |
+    | Caches the result of parsing incoming query strings to boost performance on subsequent requests.
+    |
+    */
 
+    'query_cache' => [
         /*
-         * The name of the cache item for the schema cache.
+         * Setting to true enables query caching.
          */
-        'key' => env('LIGHTHOUSE_CACHE_KEY', 'lighthouse-schema'),
+        'enable' => env('LIGHTHOUSE_QUERY_CACHE_ENABLE', true),
```

Comment:
I wonder if this should be disabled locally, like the schema option.

Reply:
I would be in favour of this (cached queries in dev have bitten me before 😛 )

Reply:
:100:

Reply:
Updated to match other cache setting in [disable query cache locally](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7434/commits/7dda5341e2e5a8ad3fc6c60d76af0084059fa29b)